lib/swaps/SwapClientManager.ts

```diff
@@ -85,6 +85,7 @@ class SwapClientManager extends EventEmitter {
           this.swapClients.set(currency.id, this.raidenClient);
         }
       });
+      await this.raidenClient.updateCapacity();
```

Comment:
I'm a little confused as to why this line is necessary, shouldn't https://github.com/ExchangeUnion/xud/blob/beb238da6dfc102cf35138f2f75d9dcf9e292c9a/lib/swaps/SwapClient.ts#L61 take care of this by calling `updateCapacity` immediately the first time that we set the timer? If you're not sure and it's not apparent after a few minutes we can maybe investigate in a future issue/pr.

If we drop this line, we can go back to making `updateCapacity` `protected` instead of `public`.

Reply:
This was necessary because the first `updateCapacity` calls happens when the swap client has been initialized and `ConnectionVerified` status is set. In the case of Raiden by that time we had not yet set the associated `tokenAddreses`.

I didn't like the fact that we made `updateCapacity` public so I moved the initial setting of `tokenAddresses` into the `init` function of Raiden.